samples/src/components/MediaDevices.jsx

```diff
@@ -70,13 +70,15 @@ export default (props) => {
         </Col>
         <Col>
           <Button
+            variant={audioStatus.muted?"danger" : 'primary'}
```

Comment:
muted/unmuted = "danger"/"primary" ?

Reply:
@bbaldino in bootstrap, different `variant`s are mapped with different specified css format. In brief, `danger` and `primary` here mean different button color.

Reply:
ha ha just for UI button to be red or blue

Reply:
gotcha :)